example-schema.ts

```diff
@@ -12,22 +12,20 @@ export type Movie = {
   runningTime: number;
 };
 
-export const movieSchema = CollectionSchema.define<Movie>("movies").indexedWith(
-  (mapping) => ({
-    exactTitle: mapping.Exact("title"),
-    title: mapping.Match(["title"], {
-      tokenFilters: [downcase, ngram({ tokenLength: 3 })],
-      tokenizer: standard,
-    }),
-    year: mapping.Range("year"),
-    runningTime: mapping.Range("runningTime"),
-    allTextDynamicMatch: mapping.DynamicMatch({
-      tokenFilters: [downcase],
-      tokenizer: ngram({ tokenLength: 3 }),
-    }),
-    allTextFieldDynamicMatch: mapping.FieldDynamicMatch({
-      tokenFilters: [downcase],
-      tokenizer: ngram({ tokenLength: 3 }),
-    }),
-  })
-);
+export async function loadMovieSchema() {
+  const moviesSchema = await new Promise<string>((resolve, reject) => {
+    readFile(join(__dirname, "./movies.schema.json"), (error, buffer) => {
+      if (error) {
+        return reject(error);
+      }
+
+      resolve(buffer.toString());
+    });
+  });
+
```

Comment:
Isn't this step redundant given that the schema is stored in the collection itself? I'm confused.

Reply:
I liked the ability to do `stash.loadCollection("movies")` - is that possible?

Reply:
Good point, I suppose the only example that actually needs this is the create collection one. I'll update everything.